README.md

```diff
@@ -6,6 +6,10 @@ operating systems (and architectures).
 
 It requires [`nix` to be installed](https://nixos.org/download.html).
 
+Once you have `nix` installed you can check that everything is working correctly using:
+* Make sure your user is a member of the `nixbld` group using `groups | grep nixbld`.
```

Comment:
I don't think this is necessary? You only need to be a trusted user. The `nixbuild` group contains the build user only. E.g. the users utilized by nix to build stuff.

Reply:
Could we then add a small paragraph to guide users on checking if they're a trusted Nix user? Is there some one-line command to verify this, rather than manually inspecting `nix.conf`?

Reply:
I have no idea. I don't even remember why I thought that was needed.

Reply:
```
grep 'trusted-users' /etc/nix/nix.conf | awk -v user="$(whoami)" '{for(i=1;i<=NF;i++) if($i == user) {f=1; print "Trusted ✅"; exit}} END {if(!f) print "Untrusted ❌"}'
```

Reply:
@yvan-sraka I think you want to use `nix show-config`, as `/etc/nix/nix.conf` may be superseded by the xdg one. It's rather annoying.